packages/http/src/Responses/Invalid.php

```diff
@@ -24,10 +26,14 @@ final class Invalid implements Response
         get => get(Validator::class);
     }
 
+    /**
+     * @param class-string|null $targetClass
+     */
     public function __construct(
         Request $request,
```

Comment:
I think we should be able to use `reflect($reuqest)` in order to not pass around the target class?